src/metakb/cli.py

```diff
@@ -14,178 +14,414 @@
 
 
-def echo_info(msg: str) -> None:
+def _echo_info(msg: str) -> None:
     """Log (as INFO) and echo given message.
 
     :param msg: message to emit
     """
     click.echo(msg)
-    logger.info(msg)
+    _logger.info(msg)
 
 
-@click.command()
-@click.option(
-    "--db_url",
-    default="bolt://localhost:7687",
-    help=(
-        "URL endpoint for the application Neo4j database. Can also be provided via environment variable METAKB_DB_URL, which takes priority."
-    ),
+def _help_msg(msg: str = "") -> None:
+    """Handle invalid user input.
+
+    :param msg: Error message to display to user.
+    """
+    ctx = click.get_current_context()
+    _logger.fatal(msg)
+
+    if msg:
+        click.echo(msg)
+    else:
+        click.echo(ctx.get_help())
+
+    ctx.exit()
+
+
+@click.group()
+def cli() -> None:
+    """Manage MetaKB data.
+
+    To reset the graph, prepare normalizers if unavailable, invalidate cached data, then
+    load MetaKB data:
+
+    \b
+        $ metakb clear-graph
+        $ metakb check-normalizers || metakb load-normalizers
+        $ metakb update --refresh_source_caches
+
+    Other commands are available for more granular control over the update process.
+    """  # noqa: D301
+
+
+_normalizer_db_url_description = "URL endpoint of normalizer database. If not given, the individual normalizers will revert to their own defaults."
+_neo4j_db_url_description = "URL endpoint for the application Neo4j database."
+_neo4j_creds_description = "Username and password to provide to application Neo4j database. Format as 'username:password'."
+
+
+@cli.command()
+@click.option("--normalizer_db_url", "-u", help=_normalizer_db_url_description)
+@click.argument(
+    "normalizer",
+    type=click.Choice(list(NormalizerName), case_sensitive=False),
+    nargs=-1,
 )
+def check_normalizers(
+    db_url: str | None, normalizer: tuple[NormalizerName, ...]
+) -> None:
+    """Perform basic checks on DB health and table population for normalizers. Exits with
+    status code 1 if >= 1 DB schema is uninitialized or critical tables appear empty for one
+    or more of the concept normalizer services.
+
+    \b
+        $ metakb check-normalizers
+        $ echo $?
+        1  # indicates failure
+
+    To select specific normalizer services, provide one or more arguments:
+
+        $ metakb check-normalizers therapy disease
+
+    Specific failures and descriptions are logged at level ERROR.
+    \f
+    :param db_url: URL endpoint for normalizer databases. Overrides defaults or env vars
+        for each normalizer service.
+    :param normalizer: tuple (possibly empty) of normalizer names to check
+    """  # noqa: D301
+    if not check_normalizer_health(db_url, normalizer):
+        _logger.warning("Normalizer check failed.")
+        click.get_current_context().exit(1)
+    _logger.info("Normalizer check passed.")
+
+
+@cli.command()
+@click.option("--normalizer_db_url", "-n", help=_normalizer_db_url_description)
```

Comment:
Since Dynamo takes a single URL but Postgres requires different libpq-style URLs that include normalizer-specific database names, I don't think it's really feasible to support postgres connections this way yet. Thought about doing something that could take a base postgres URL and add database names on top but it's probably not worth the work right now.